src/main/java/uk/gov/digital/ho/hocs/info/domain/exception/RestResponseEntityExceptionHandler.java

```diff
@@ -40,6 +40,12 @@ public ResponseEntity handle(ApplicationExceptions.EntityCreationException e) {
         return new ResponseEntity<>(e.getMessage(), INTERNAL_SERVER_ERROR);
     }
 
+    @ExceptionHandler(ApplicationExceptions.UserAlreadyExistsException.class)
+    public ResponseEntity handle(ApplicationExceptions.UserAlreadyExistsException e) {
+        log.warn("ApplicationExceptions.UserAlreadyExistsException: {}, Event: {}", e.getMessage(), value(EVENT, e.getEvent()));
```

Comment:
```suggestion
        log.warn("ApplicationExceptions.UserAlreadyExistsException: {}", e.getMessage(), value(EVENT, e.getEvent()));
```
The `value(EVENT, e.getEvent())` is used by Kibana - IntelliJ just moans about it.

Reply:
I have made this change